### ISSUE TITLE
Update Competition Dash All Impact #s filter for flagged but RB count does not. 

### ISSUE THREAD

**Issue by jessleenyc** [DoSomething/quasar#268]
Competitions team would like to filter out flagged rbs for impact #s but include flagged rbs for RB counts. 

Specifically look at the competitions summary table.

**Comment by nmccormick619** [comment]
@jessleenyc I made these changes last week. I just took a second look and it seems like everything is good. Let me know if you see anything that stands out still!

**Comment by jessleenyc** [comment]
@nmccormick619 okay so to confirm, even when i put the flagged filter on, the impact number won't be affected?

**Comment by nmccormick619** [comment]
@jessleenyc I don't see a flagged filter on that look

**Comment by jessleenyc** [comment]
@nmccormick619 ah sorry! let me clarify. yesterday, I removed the flagged filter that was on that dashboard because the impact team would rather have a total 'count of competition reportbacks' but they want to continue having impact numbers filter out flagged items. so, I wanted to make sure that when I removed the flagged filter, that the impact numbers weren't affected since they DO need to exclude flagged.
